tests/unit/migrations/test-class-track-upe-status.php

```diff
@@ -59,21 +59,6 @@ public function test_track_enabled_on_upgrade() {
 		$this->assertSame( '1', get_option( Track_Upe_Status::IS_TRACKED_OPTION ) );
 	}
 
-	public function test_track_disabled_on_upgrade() {
-		update_option( WC_Payments_Features::UPE_FLAG_NAME, 'disabled' );
-
-		Track_Upe_Status::maybe_track();
-
-		$this->assertEquals(
-			[
-				'wcpay_upe_disabled' => [],
-			],
-			Tracker::get_admin_events()
-		);
-
-		$this->assertSame( '1', get_option( Track_Upe_Status::IS_TRACKED_OPTION ) );
-	}
-
```

Comment:
No need to check for `wcpay_upe_disabled` anymore since UPE is always disabled now